internal/cli/universal_login_customize.go

```diff
@@ -389,7 +402,17 @@ func (h *webSocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				promptToFetch.Language,
 			)
 			if err != nil {
-				h.display.Errorf("failed to fetch custom text for prompt: %v", err)
+				h.display.Errorf("Failed to fetch custom text for prompt: %v", err)
+
+				errorMsg := webSocketMessage{
+					Type:    errorMessageType,
+					Payload: []byte(fmt.Sprintf(`{"error":%q}`, err.Error())),
```

Comment:
Should we be using a struct to represent the error, and then JSON-encode it?

Reply:
Yep, we should. I plan to do that in a follow up PR and refactor the entire webSocketMessage so we can add custom JSON marshalling and unmarshalling and reuse the payload accordingly. This will prevent us from needing to unmarshal twice the message when we received it through the websocket as we're doing now.

Reply:
Here it is @Widcket -> https://github.com/auth0/auth0-cli/pull/880